packages/communication-react/review/communication-react.api.md

```diff
@@ -714,6 +742,9 @@ export type DefaultChatHandlers = {
     onLoadPreviousChatMessages: (messagesToLoad: number) => Promise<boolean>;
 };
 
+// @public (undocumented)
+export const defaultControlButtonStyle: IButtonStyles;
```

Comment:
nit: This should be a function that returns a freshly cloned object, so that Contoso can't (unintentionally) modify the value of the constant for all buttons.